exercises/reverse-string/example.jl

```diff
@@ -0,0 +1,12 @@
+function myreverse(phrase::String)
+    chars = []
+    for i in phrase
+        push!(chars, i)
+    end
```

Comment:
`c` would be a better name. `i` is typically an integer counter.

You could also use `split("abc", "") to create an array from a string.